### DISCUSSION TITLE
Jenkin build failing  dev/Build/Core/Yarn/

### DISCUSSION THREAD

**Discussion by naveenhegde22** [Sunbird-Ed/Community#125] [Sunbird ED configuration discussion] [answered]
Jenkins build failing with compilation error 

repo:- https://github.com/project-sunbird/sunbird-lms-jobs.git
tag:-release-2.6.0
error:-
[[1;31mERROR[m] Failed to execute goal [32morg.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile[m [1m(default-compile)[m on project [36mcommon-util[m: [1;31mCompilation failure[m: Compilation failure: 
[[1;31mERROR[m] /var/lib/jenkins/workspace/Build/Core/Yarn/modules/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/datasecurity/impl/DefaultEncryptionServivceImpl.java:[127,27] error: cannot find symbol
[[1;31mERROR[m]  package sun.misc
[[1;31mERROR[m] /var/lib/jenkins/workspace/Build/Core/Yarn/modules/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/datasecurity/impl/DefaultDecryptionServiceImpl.java:[100,43] error: cannot find symbol

**Comment by beepdot** [comment, marked as answer]
This is a very old github tag. Can you use newer tags?
